.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  push:
+    tags:
+      - '**'
+jobs:
+  tests:
+    uses: ./.github/workflows/tests.yml
+  release:
+    runs-on: ubuntu-latest
+    needs: tests
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.node-version'
+          # Configure NPM to point to Github's package registry.
+          registry-url: https://npm.pkg.github.com/
+
+      - name: Install deps
+        uses: bahmutov/npm-install@v1
+
+      # Publish the packages on Github's package registry for internal CZI use. These should
+      # already be published to npm before we get here.
+      - name: Publish packages to the Github registry
+        run: npx lerna publish from-package --yes
```

Comment:
`from-package` means this publishes the **_current_** version numbers, instead of getting new ones.